platform/ios/platform/macos/INSTALL.md

```diff
@@ -23,7 +23,7 @@ Before building, follow these steps to install prerequisites:
    # To install the CMake documentation
    brew list cmake-docs || brew install cmake-docs
    ```
-1. Install [xcpretty](https://github.com/supermarin/xcpretty) (optional, used for prettifying command line builds):
+1. Install [xcpretty](https://github.com/supermarin/xcpretty) (used for prettifying command line builds):
```

Comment:
Thanks for the update, but please revert this. This seems to be a bug and should be fixed. Your change is mainly a workaround.

I'm still surprised though why the CI succeeds.

Reply:
Probably because xcpretty is installed on the CI runner.